code/API_definitions/device-status.yaml

```diff
@@ -466,24 +473,36 @@ components:
     RoamingStatusResponse:
       type: object
       required:
+        - lastChecked
         - roaming
       properties:
+        lastChecked:
+          $ref: "#/components/schemas/LastChecked"
         roaming:
           $ref: "#/components/schemas/ActiveRoaming"
         countryCode:
           $ref: "#/components/schemas/CountryCode"
         countryName:
           $ref: "#/components/schemas/CountryName"
 
+    LastChecked:
```

Comment:
For me the name is not clear enough, what was checked? Maybe the name should follow the device-status pattern: LastStatusTime

Reply:
Since the field represents the time when the status was last checked, using "checked" instead of "status" would indeed be more appropriate and clearer in conveying the purpose of the field . However,  I'm open to updating the field name to either "LastCheckedTime" or "LastStatusTime". I'd like to hear feedback from others on which option the team prefers.

Reply:
I also like "lastStatusTime". It is similar to what has been done in DeviceLocation with "lastLocationTime". https://github.com/camaraproject/DeviceLocation/blob/main/code/API_definitions/location-retrieval.yaml#L239